components/script/dom/document.rs

```diff
@@ -2145,8 +2141,8 @@ impl Document {
             // for the interim frames where we are deciding whether this rAF
             // is considered spurious, we need to ensure that the layout
             // and compositor *do* tick the animation.
-            self.window
-                .force_reflow(ReflowGoal::Full, ReflowReason::RequestAnimationFrame, None);
+            println!("Reflow will not be triggered");
```

Comment:
Should this be a logging call instead?

Reply:
I had removed this `println!` locally, but neglected to commit my change. I'll remove it.